tests/test_contract_resources.py

```diff
@@ -16,7 +16,7 @@
 
 from profitbricks.client import ProfitBricksService
 
-from .helpers import configuration
+from helpers import configuration
```

Comment:
Why changing the relative import?

Reply:
When executing the unittests we run something like this:
python -m unittest discover tests test_snapshot.py

If we run that from the command line the following output will happen:
'ImportError: attempted relative import with no known parent package'

The reasons for why this happens are explained here: https://stackoverflow.com/questions/14132789/relative-imports-for-the-billionth-time